### DISCUSSION TITLE
how to install newman-reporter-html & newman-reporter-htmlextra

### DISCUSSION THREAD

**Discussion by paul-z-2020** [Homebrew/discussions#3138] [Everyday usage]
Hi everyone.

I used Brew to install Newman with no issues. However, when I tried to install the reporter packages, I got:

```
Warning: No available formula with the name "newman-reporter-html".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
```

I then used "npm install -g" to install the 2 reporter packages.
When I tried to run Newman, I got:

```
newman: could not find "htmlextra" reporter
  ensure that the reporter is installed in the same directory as newman
  please install reporter using npm
```

How to install the packages and have them work with the existing Newman installed by Brew?

Thanks in advance!

Paul

**Comment by paul-z-2020** [comment]
It worked when I moved /opt/homebrew/lib/node_modules/<package names> to /opt/homebrew/Cellar/newman/5.3.2/libexec/lib/node_modules/

Is this the correct solution?

Cheers,

Paul

**Comment by adisheshbs** [comment]
Steps to fix this once in for all:

Pre-requisites: nodejs and npm packages are installed successfully. 

1. Uninstall newman package installed using Brew. `brew uninstall newman`
2. Install newman globally using npm -> run `sudo npm install -g newman`
3. Install newman html reporter globally again using npm only -> `npm install -g newman-reporter-htmlextra`
4. Now it should work perfectly and generate html reports. 

Hope it helps!
